lib/parsers.js

```diff
@@ -83,9 +95,11 @@ exports.parseError = function (err, agent, cb) {
       if (match) error.exception.code = match[1]
     }
 
-    var props = stackman.properties(err)
-    if (props.code) delete props.code // we already have it directly on the exception
-    if (Object.keys(props).length > 0) error.exception.attributes = props
+    if (captureAttributes) {
+      var props = stackman.properties(err)
+      if (props.code) delete props.code // we already have it directly on the exception
+      if (Object.keys(props).length > 0) error.exception.attributes = props
+    }
```

Comment:
my personal value of "don't have single line brace-less conditionals" is battling it out with my personal value of "leave code as you found it" with [no clear winner](https://www.youtube.com/watch?v=t-dK9996Ks4&has_verified=1&t=0m37s)

Reply:
Heh, I went through the same debate, albeit without a star trek reference.